helper/impls/helper-impl-111/src/main/java/com/linkedin/avroutil1/compatibility/avro111/codec/AliasAwareSpecificData.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2022 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+
+package com.linkedin.avroutil1.compatibility.avro111.codec;
+
+import com.linkedin.avroutil1.compatibility.ClassLoaderUtil;
+import org.apache.avro.Schema;
+import org.apache.avro.specific.SpecificData;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class AliasAwareSpecificData extends SpecificData {
```

Comment:
I know this duplication is due to the trickiness of this build, and I am not necessarily expecting an improvement as part of this PR, but it'd be really nice to find a way to reduce this duplication... Macros, maybe?

Reply:
eliminated some duplication